docs/layers/cpu-grid-layer.md

```diff
@@ -168,6 +168,11 @@ larger than the elevationUpperPercentile will be hidden.
 Filter cells and re-calculate elevation by `elevationLowerPercentile`. Cells with elevation value
 smaller than the elevationLowerPercentile will be hidden.
 
+##### `colorScaleType` (String, optional)
+
+* Default: 'quantize'
+
+Color scale type, default is set to 'quantize'. Supported Values are 'quantize', 'linear', 'quantile' and 'ordinal'.
```

Comment:
define the props as, `scaling function used to determine the color of the grid cell, default value is ...`